native/src/sig.rs

```diff
@@ -7,28 +10,17 @@ use ssb_crypto::{Keypair, NetworkKey as AuthKey, PublicKey, Signature};
 // sign: (keys: obj | string, hmac_key?: string, o: obj) => string
 pub fn neon_sign_obj(mut cx: FunctionContext) -> JsResult<JsObject> {
   // FIXME: detect `curve` from keys.curve or from u.getTag and validate it
-  let args_length = cx.len();
-  if args_length < 2 {
+  let argc = cx.len();
```

Comment:
Pardon my silly name change, I just wanted the `let (index, ord) = ` below to fit on one line :)

Reply:
Totally acceptable